segmentstore/server/src/test/java/io/pravega/segmentstore/server/SegmentStoreMetricsTests.java

```diff
@@ -161,30 +161,30 @@ public void testThrottlerMetrics() {
 
     @Test
     public void testCacheManagerMetrics() {
-        int storedBytes = 10000;
-        int usedBytes = 1000;
-        int allocatedBytes = 100;
+        long storedBytes = 10000;
+        long usedBytes = 1000;
+        long allocatedBytes = 100;
         int generationSpread = 10;
-        int managerIterationDuration = 1;
+        long managerIterationDuration = 1;
 
         int containerId = new Random().nextInt(Integer.MAX_VALUE);
         @Cleanup
         SegmentStoreMetrics.CacheManager cache = new SegmentStoreMetrics.CacheManager();
         cache.report(new CacheState(storedBytes, usedBytes, 0, allocatedBytes, storedBytes), generationSpread, managerIterationDuration);
 
-        assertEquals(storedBytes, (int) MetricRegistryUtils.getGauge(MetricsNames.CACHE_STORED_SIZE_BYTES).value());
-        assertEquals(usedBytes, (int) MetricRegistryUtils.getGauge(MetricsNames.CACHE_USED_SIZE_BYTES).value());
-        assertEquals(allocatedBytes, (int) MetricRegistryUtils.getGauge(MetricsNames.CACHE_ALLOC_SIZE_BYTES).value());
+        assertEquals(storedBytes, (long) MetricRegistryUtils.getGauge(MetricsNames.CACHE_STORED_SIZE_BYTES).value());
```

Comment:
Please wait sufficient time before assertion (using assert utility tools), otherwise when the testing env is slow, the test will fail again randomly.

Reply:
Same response as below.